components/build/tekton-chains/chains-controller-deployment.yaml

```diff
@@ -8,7 +8,10 @@ spec:
     spec:
       containers:
       - name: tekton-chains-controller
-        image: ghcr.io/hacbs-contract/chains/controller:cd2d8c2c9fea3902248dd811913e7ea4fa73d639
+        # Use a nightly-build image because the current latest version, 0.8.0, contains
+        # several bugs that make Chains unusable for our use case. Once version 0.9.0
+        # is published, change this reference to that.
+        image: gcr.io/tekton-nightly/github.com/tektoncd/chains/cmd/controller@sha256:9b4bef2e9d617dddfc0243b78db2ce8b0f176460bc408bd28e5b31cba79bdece
```

Comment:
I initially wanted this to use the `:latest` tag so we would catch breaking upstream issues right away. But I've talked myself out of it because it would be a terrible development experience. Also, the `:latest` tag causes ambiguity on which image will actually be used.

We can always update this reference when there's a need to (a new bug fix, a new release, etc).